### DISCUSSION TITLE
precision-recall in survival analysis

### DISCUSSION THREAD

**Discussion by ad2n15** [sebp/scikit-survival#262] [Q&A]
Hi,

The package provides C-index, IBS, and time dependant AUC-ROC for the evaluation of a model. How could I do calculate precision, recall, calculate AUC and plot precision-recall curve?

Kind regards

**Comment by sebp** [comment]
An estimator of the precision-recall curve is currently not implemented.

Personally, I haven't heard about it being a common performance measure 
for right censored data.
